meeting-notes/2018/201Y8-11-12--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,206 @@
+ - continue implementing the current QUIC draft
+
+@kevina (might not be able to attend)
+  - Done:
+    - Review p.r. to switch to new commands library.  Almost all the commands are now using the new commands library except for `ipfs ls`.
+  - Next:
+    - Redo `--cid-base` p.r. now that the command line conversion is in, may make things simpler
+
+@schomatis (not attending)
+ - Done/In Progress:
+   - Review `ipfs/docs` PRs (the repo hasn't been active for some months now).
+   - Write content for a concept document about files in IPFS (https://github.com/ipfs/docs/issues/133).
+   - Merge DAG walker (https://github.com/ipfs/go-ipld-format/pull/39).
+   - Review HAMT directory refactor PR (https://github.com/ipfs/go-unixfs/pull/42).
+   - Review trickle DAG refactor PR (https://github.com/ipfs/go-unixfs/pull/10).
+ - Next (not necessarily this week):
+   - With the DAG walker merged finish up the DAG reader refactor using it (https://github.com/ipfs/go-unixfs/pull/12).
+   - Write mid-level document on the `ipfs files` API (https://github.com/ipfs/go-ipfs/pull/5052)
+   - MFS interface (https://github.com/ipfs/go-mfs/issues/3).
+
+@magik6k (not attending)
+ - Done:
+   - Running datastore benchmarks right now
+   - (Almost) have consensus on go-ipfs-files refactor
+   - Did some reviews
+ - Blocked:
+   - No, but will need some reviews this week
+ - Next:
+   - Run all the benchmarks, on all the things
+   - Finish go-ipfs-files stuff
+   - Write a doc on datastore performance
+
+@eingenito
+  - Done:
+    - Worked on [2019 Planning Doc](https://docs.google.com/document/d/11zw8H8NjUzMM5CJ1y9lpvDxmVrJ6SAOxsRsiLPZxDaI/edit#heading=h.3o5e0g37o69l)
+    - Did [screencast of same](https://drive.google.com/open?id=1MHzaHujcYfOhQEOd0PseViTUkqm6Ap5D)
+    - Worked on bitswap with Hannah
+    - Investigated benchmarking - so many options
+  - Next
+    - OKR projections
+    - Benchmarking - just choose one
+    
+  
+@warpfork
+  - Done:
+    - Worked on [2019 Planning Doc](https://docs.google.com/document/d/11zw8H8NjUzMM5CJ1y9lpvDxmVrJ6SAOxsRsiLPZxDaI/edit)
+    - There have been some discussions on how representations of serial zeros and nulls and whatnot should work: https://github.com/polydawn/refmt/issues/42
+    - https://github.com/ipld/specs/pull/72 merged woo (thank you mikeal)
+    - go-ipld-prime has a MutableNode interface now and it has one impl; a fluent wrapper interface layer for convenience; some tests
+    - did some dives on merkleCI agent models across platforms (punted to victorb for review)
+  - Blocked: n/a
+  - Next:
+    - flesh out (way) more tests in go-ipld-prime, finish several todos for recursive mutables
+    - build a first draft of traversers and zippers in go-ipld-prime
+    
+@bigs
+  - Done:
+    - Local kubernetes cluster with daemon images!
+    - Code review
+    - SECIO spec first pass
```

Comment:
@bigs I hope someone told you about https://github.com/auditdrivencrypto/secure-channel/blob/master/prior-art.md#ipfss-secure-channel, a long time write and reverse spec done by @dominictarr.

Reply:
Also, https://github.com/libp2p/js-libp2p/tree/master/examples/encrypted-communications should be useful